includes/scripts.php

```diff
@@ -21,6 +21,7 @@ function wprss_register_scripts()
         wp_localize_script( 'wprss-admin-addon-ajax', 'wprss_admin_addon_ajax', array(
             'please_wait'   =>  __( 'Please wait ...', WPRSS_TEXT_DOMAIN )
         ));
+        wp_register_script( 'lorem-embed-script', 'https://embed.asklorem.com/load.js', array('wprss-admin-addon-ajax'), null, true );
```

Comment:
i think we want the array to be empty since our script doesn't depend on it (see w3tc code)

wp_register_script( 'lorem-embed-script', 'https://embed.asklorem.com/load.js' , array(), null, true );